packages/nvidia_nat_core/src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -775,9 +778,20 @@ async def get_single(response: Response, request: Request):
                         http_connection=request,
                         user_authentication_callback=self._http_flow_handler.authenticate) as session:
 
-                    result = await generate_single_response(None, session, result_type=result_type)
-                    add_context_headers_to_response(response)
-                    return result
+                    try:
+                        result = await generate_single_response(None, session, result_type=result_type)
+                        add_context_headers_to_response(response)
+                        return result
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)